client/js/widgets/spinner.js

```diff
@@ -8,7 +8,7 @@ class Spinner extends Widget {
       typeClasses: 'widget spinner',
       clickable: true,
 
-      options: [ 1, 2, 3, 4, 5, 6 ],
+      options: [],
```

Comment:
Please redo this so you don't change the actual default but instead just change what the JSON editor inserts.

Reply:
Fixed in latest commit.